### PR TITLE
Feature/unified pipelines

### PR DESCRIPTION
**Pull request by mitchdenny**
This PR enables unified pipelines on the storage client libraries. Once this is merged we'll apply the same approach to all the libraries we are shipping as a preview next week. Because we use the same pipelines for management plane as well - I also made the changes as necessary for the advisor code base.

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/advisor/ci.yml [hunk 1]
```diff
@@ -21,14 +35,11 @@ pr:
...
     - sdk/advisor/
 
-resources:
-  repositories:
-    - repository: azure-sdk-tools
-      type: github
-      name: Azure/azure-sdk-tools
-      endpoint: azure
 
 jobs:
```
Comment (danieljurek) [inline sdk/advisor/ci.yml]: This should probably be `stages`

Comment (mitchdenny) [reply sdk/advisor/ci.yml]: Whoops! Yep - that always catches me out when I am rolling out these changes. Thanks!

**Pull request merged**